src/main.ts

```diff
@@ -181,8 +205,61 @@ export default function main() {
     return res;
   }
 
+  async function initFeedbackPrompting(
+    userId?: User["userId"],
+    requestCallback?: FeedbackPromptCallback,
+  ) {
+    checkKey();
+    if (ablyClient) err("Feedback prompting already initialized. Use reset() first.");
+    if (persistUser) {
+      userId = getSessionUser();
+    } else if (!userId) {
+      err("No userId provided and persistUser is disabled");
+    }
+    const res = await request(`${getUrl()}/feedback/prompting-status`, {
+      userId,
+    });
+    log(`feedback prompting status sent`, res);
+    const body: { success: boolean, channel?: string } = await res.json()
+    if (!body.success || !body.channel) {
+      log(`feedback prompting not enabled`);
+      return res;
+    }
+
+    log(`feedback prompting enabled`);
+    const actualCallback = requestCallback || showFeedbackPrompt;
+    ablyClient = await openAblyConnection(`${getUrl()}/feedback/prompting-auth`, userId, body.channel, (data) => {
+      if (typeof data?.question !== "string" ||
+        typeof data?.showAfter !== "number" ||
+        typeof data?.showBefore !== "number") {
+        err(`invalid feedback prompt message received`, data);
+      } else {
+        log(`feedback prompt received`, data);
+        actualCallback({
+          question: data.question,
+          showAfter: new Date(data.showAfter),
+          showBefore: new Date(data.showBefore),
+        });
+      }
+    }, debug)
+
+    log(`feedback prompting connection established`);
+    return res;
+  }
+
+  function showFeedbackPrompt(prompt: FeedbackPrompt) {
+    // nothing to do here yet
+    log(`showing feedback prompt`, prompt);
+  }
```

Comment:
lets just remove this for now? no need to have that in here. It's totally fine to ship prompting without the embeddable widget.